appengine/genetics/js/genetics.js

```diff
@@ -237,14 +514,506 @@ Genetics.init = function() {
+        x: 250,
+        y: 300,
+        direction: Math.PI / 2,
+        size: 3
+      },
+      {
+        // Create same size mouse.
+        id: 3,
+        playerId: 3,
+        x: 300,
+        y: 200,
+        direction: Math.PI / 2,
+        size: 2
+      },
+      {
+        // Create smaller mouse.
+        id: 4,
+        playerId: 1,
+        x: 250,
+        y: 100,
+        direction: Math.PI / 2,
+        size: 1
+      },
+      {
+        // Create smaller mouse.
+        id: 5,
+        playerId: 2,
+        x: 150,
+        y: 300,
+        direction: Math.PI / 2,
+        size: 1
+      },
+      {
+        // Create smaller mouse.
+        id: 6,
+        playerId: 3,
+        x: 100,
+        y: 200,
+        direction: Math.PI / 2,
+        size: 1
+      }
+    ];
+    goog.array.shuffle(startingMice);
+  } else if (BlocklyGames.LEVEL <= 8) {
+    startingMice = [
+      {
+        // Create player mouse.
+        id: 0,
+        playerId: 0,
+        x: 200,
+        y: 200,
+        direction: -Math.PI / 2,
+        size: 2,
+        fertility: 5
+      },
+      {
+        // Create bigger mouse.
+        id: 1,
+        playerId: 1,
+        x: 150,
+        y: 100,
+        direction: Math.PI / 2,
+        size: 3,
+        sex: Genetics.Mouse.Sex.MALE
+      },
+      {
+        // Create bigger mouse.
+        id: 2,
+        playerId: 2,
+        x: 250,
+        y: 300,
+        direction: Math.PI / 2,
+        size: 3,
+        sex: Genetics.Mouse.Sex.FEMALE
+      },
+      {
+        // Create bigger mouse.
+        id: 3,
+        playerId: 3,
+        x: 300,
+        y: 200,
+        direction: Math.PI / 2,
+        size: 3,
+        sex: Genetics.Mouse.Sex.MALE
+      },
+      {
+        // Create bigger mouse.
+        id: 4,
+        playerId: 1,
+        x: 250,
+        y: 100,
+        direction: Math.PI / 2,
+        size: 3,
+        sex: Genetics.Mouse.Sex.FEMALE
+      },
+      {
+        // Create same size mouse.
+        id: 5,
+        playerId: 2,
+        x: 150,
+        y: 300,
+        direction: Math.PI / 2,
+        size: 2,
+        sex: Genetics.Mouse.Sex.FEMALE
+      },
+      {
+        // Create same size mouse.
+        id: 6,
+        playerId: 3,
+        x: 100,
+        y: 200,
+        direction: Math.PI / 2,
+        size: 2,
+        sex: Genetics.Mouse.Sex.MALE
+      }
+    ];
+    goog.array.shuffle(startingMice);
+  } else {  // BlocklyGames.LEVEL >= 9
+    startingMice = [];
+    var mouseId = 0;
+    for (var playerId = 0; playerId < 4; playerId++) {
+      for (var i = 0; i < 2; i++) {
+        startingMice.push({
+          id: mouseId++,
+          sex: i % 2 == 0 ? Genetics.Mouse.Sex.MALE :
+              Genetics.Mouse.Sex.FEMALE,
+          playerId: playerId
+        });
+      }
+    }
+    goog.array.shuffle(startingMice);
+  }
+  for (var i = 0, mouseStats; mouseStats = startingMice[i]; i++) {
+    var sex = (mouseStats.sex != null) ? mouseStats.sex :
```

Comment:
I agree for sex.
Fertility and aggressiveness can be 0, however.
I'm not so sure about using that approach for size because I don't want to change the size unless mouseStats is set so it looks a little weird. I could do:
`mouse.size = mouseStats.size || mouse.size;`